mule-extensions-api-persistence/src/test/resources/extension/list-of-serialized-extension-model.json

```diff
@@ -13,20 +13,22 @@
       "schemaVersion": "",
       "xsdFileName": ""
     },
-    "subTypes": [
-
-    ],
-    "importedTypes": [
-
-    ],
-    "displayModel": {
-
-    },
-    "configurations": [
-
-    ],
+    "subTypes": [],
+    "importedTypes": [],
+    "displayModel": {},
+    "configurations": [],
     "operations": [
       {
+        "errorTypes": [
```

Comment:
add a todo so that any component declaring errors, do it as a reference to the errors listed en ExtensionModel#getErrorModels()